src/components/Breadcrumb/BreadcrumbOverflowItem.jsx

```diff
@@ -0,0 +1,126 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
```

Comment:
Same thing here with refactoring to utilize sass. Please use carbon token variables where possible. This includes layout/sizing items, motion bezier curves, colors, and typography styles.